coral/src/app/features/topics/details/components/TopicDetailsResourceTabs.tsx

```diff
@@ -63,6 +78,43 @@ function TopicOverviewResourcesTabs({ currentTab, topicName }: Props) {
     },
   ];
 
+  function renderTabContent() {
+    if (isError) {
+      return (
+        <Box marginBottom={"l1"} marginTop={"l2"} role="alert">
+          <Alert type="error">
+            There was an error trying to load the topic details{" "}
+            {parseErrorMsg(error)}.
+            <br />
+            Please try again later.
+          </Alert>
+        </Box>
+      );
+    }
+
+    if (isLoading) {
+      return (
+        <Box paddingTop={"l2"} display={"flex"} justifyContent={"center"}>
+          <div className={"visually-hidden"}>Loading topic details</div>
+          <Icon icon={loading} fontSize={"30px"} />
```

Comment:
Nitpick: Could we have a skeleton view depending on the tab we are on? The little loading icon is so lonely in the white void 😭 

Probably a followup ^^

Reply:
Yes, that's a good follow up! Will create an issue.